packages/cactus-plugin-ledger-connector-quorum/src/main/typescript/plugin-ledger-connector-quorum.ts

```diff
@@ -335,6 +451,172 @@ export class PluginLedgerConnectorQuorum
     }
   }
 
+  public async invokeContractNoKeychain(
```

Comment:
@TonyRowntree Please look into it if smaller portions of this method could be extracted into common methods and then those methods could be shared between this method and the invokeContract method as well.